content/_releases/22/20.md

```diff
@@ -0,0 +1,165 @@
+
+
+### White-labeled credential management
+
+Integration projects for enterprise or retail solutions can become complex since there is a
+need to connect multiple endpoints within the company ecosystem. This complexity,
+oftentimes, dictates a certain rules of engagement like restricted access to the
+systems. On the other hand, unrestricted access to these system is essential for
+integration solution.
+
+From one side we have the need for integration and other side, the need for security.
+Often times, the team developing the integration is not the same team holding the
+access to these systems. To solve the dilemma we introduce **White-labeled credential management**
+feature.
+
+#### How does the White-labeled credential management work?
+
+As integrator you provide a unique URL to the credential holder to enter their access credentials without
+registering or logging-in to the system. The access holder is presented an interface
+to enter their credentials, verify them and save.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2220/RN2220-create-credential.png" title="OEM Create Credential" %}
+
+They can also modify the existing credentials as well.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2220/RN2220-modify-credentials.png" title="OEM Modify Credentials" %}
+
+For both cases they are presented with a simple screen to work with only the
+specified credentials.
+
+#### How to generate unique URL?
+
+There are certain preconditions before you can make this work:
+
+*   The end customer is already in the DB. This means you have to [create a user]({{site.data.tenant.apiDocsUri}}/v2/#create-a-user) for your credential holder, then [add him/her to the contract]({{site.data.tenant.apiDocsUri}}/v2/#add-a-new-member-to-the-contract's-scope) and [workspace]({{site.data.tenant.apiDocsUri}}/v2/#add-a-new-member-to-workspace) where the integration is set. All these actions you can do with API calls using administrative access.
+*   New created user must have only access to the credentials entity.
+*   Your tenant has SSO provider (OIDC/SAML) with `autoCreateUsers` set to `false` configuration.
+
+If the above preconditions are in place then here is how you can generate the URL
+to give to your credential holder.
+
+The initial URL has a form `/embedded-credentials/repoId` where the `repoId`
+would be the ID of component for which you need to set the credential. Then you
+would need to add the following query parameters to this URL:
+
+*   `workspaceId` (required) - this is the ID of the workspace where integration is set.
+*   `ssoProviderType` (optional) - values can be `SAMLProvider` or `OIDCProvider`.
+*   `ssoProviderId` (optional) - this is the ID of your SSO provider.
+
+Let us construct the URL:
+`TENANT_URL/embedded-credentials/[repoId]?workspaceId=[workspaceId]&ssoProviderType=[ssoProviderType]&ssoProviderId=[ssoProviderId]`
+
+If you do not specify `ssoProviderType` and `ssoProviderId` then they will be
```

Comment:
then they will automatically be taken from the XXXXXXXXXXXX property, but only if this is defined.


The rest adds little as we already described ssoprovider Type & ID as prerequisites